app/assets/stylesheets/application_rmc-legacy.scss

```diff
@@ -0,0 +1,40 @@
+@import 'base';
+
+// Theme designed to include a modified version of the original RMC masthead image in larger screen sizes
+
+.masthead {
+  .site-title-container {   
+    .site-title-wrapper {
+      // Remove block background behind text
+      background: none;
+    }
+
+    @media (min-width: breakpoint-max('md')) {
+      // For larger screen sizes, hide title text (except for screen readers)
+      height: 178px;
+      .site-title {
+        @include sr-only;
+        }
+    }
+  }
+
+  .background-container {
+    // Remove blur filter and center masthead image
```

Comment:
We have this ticket for not distorting the masthead image: https://culibrary.atlassian.net/browse/LP-1168. Do you know if some of these css changes you're applying in this theme could be added to the default css to address this ticket?

Reply:
Done! I also removed a similar override that you did in the Joyce theme.